### ISSUE TITLE
Cocoapods Privacy Manifest

### ISSUE THREAD

**Issue by lixiang1994** [SwiftyBeaver/SwiftyBeaver#528]
podspec
```ruby
s.resource_bundles = {"SwiftyBeaver" => 'PrivacyInfo.xcprivacy'}
```

**Comment by lixiang1994** [comment]
@skreutzberger

**Comment by skreutzberger** [comment]
Thanks for the hint! 

I released version 2.0.2 on CocoaPods. It contains now the line `s.resource_bundles = {"SwiftyBeaver" => 'PrivacyInfo.xcprivacy'}` in the podspec. Does it work for you guys?

**Comment by skreutzberger** [comment]
... and I also released 2.1.0 bringing support for OSLog API 🎉
